docs/modules/agents/scratch.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5436020b",
+   "metadata": {},
+   "source": [
+    "# Getting Started\n",
+    "\n",
+    "Agents use an LLM to determine which actions to take and in what order.\n",
+    "An action can either be using a tool and observing its output, or returning to the user.\n",
+    "\n",
+    "When used correctly agents can be extremely powerful. The purpose of this notebook is to show you how to easily use agents through the simplest, highest level API."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c6226b9",
+   "metadata": {},
+   "source": [
+    "In order to load agents, you should understand the following concepts:\n",
+    "\n",
+    "- Tool: A function that performs a specific duty. This can be things like: Google Search, Database lookup, Python REPL, other chains. The interface for a tool is currently a function that is expected to have a string as an input, with a string as an output.\n",
+    "- LLM: The language model powering the agent.\n",
+    "- Agent: The agent to use. This should be a string that references a support agent class. Because this notebook focuses on the simplest, highest level API, this only covers using the standard supported agents. If you want to implement a custom agent, see the documentation for custom agents (coming soon).\n",
+    "\n",
+    "**Agents**: For a list of supported agents and their specifications, see [here](agents.md).\n",
+    "\n",
+    "**Tools**: For a list of predefined tools and their specifications, see [here](tools.md)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "d01216c0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain import OpenAI, SQLDatabase, SQLDatabaseChain\n",
+    "from langchain.chains.sql_database.base import SQLDatabaseChain"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef965094",
+   "metadata": {},
+   "source": [
+    "First, let's load the language model we're going to use to control the agent."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "0728f0d9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "db = SQLDatabase.from_uri(\"sqlite:////Users/BrendanLeahey/Desktop/Independent/langchain/notebooks/Chinook.db\")\n",
+    "llm = OpenAI(temperature=0, openai_api_key=\"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\")"
```

Comment:
here

Reply:
@bpleahey messaged you offline about this

@hwchase17 favor to ask, is there a way to nuke a branch + history? thanks in advance!